### DISCUSSION TITLE
Group Alerts with State

### DISCUSSION THREAD

**Discussion by ethernoy** [grafana/grafana#58555] [Alerting]
I noticed the alert in Grafana 9 supports grouping with label like alertname. Is it also possible to group the alert event with its state?

In other words, can I configure Grafana to send firing and resolved events in separate alert message?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
